solutions/회의실배정.js

```diff
@@ -0,0 +1,31 @@
+const INPUT_PATH = "../inputs/회의실배정.txt"
+
+const fs = require("fs");
+const filePath = process.platform === "linux" ? "/dev/stdin" : INPUT_PATH;
+const input = fs.readFileSync(filePath).toString().trim().split("\n");
+
+function solution(input){
+  const n = parseInt(input[0]);
+  const meetings = [];
+
+  for (let i = 1; i <= n; i++) {
+    const [start, end] = input[i].split(" ").map(Number);
+    meetings.push({ start, end });
+  }
```

Comment:
```suggestion
  for (let i = 1; i <= n; i++) {
    const [start, end] = input[i].split(" ").map(Number);
    
    meetings.push({ start, end });
  }
```

저는 선언문과, 비즈니스 로직은 한줄 띄어서 작성하는 편 인데 이부분은 개인적인 스타일 같습니다!

Reply:
해당 반복문에서는 input[i]로 배열의 값들을 가져오고 있는데, 그게 주 목적이라면 `forEach`나, `for...of`를 사용할 수도 있겠네요!